nuxeo-core/nuxeo-core-event/src/main/java/org/nuxeo/ecm/core/work/WorkManagerImpl.java

```diff
@@ -649,6 +649,12 @@ protected void beforeExecute(Thread t, Runnable r) {
                 queuing.workReschedule(queueId, work);
                 throw new RejectedExecutionException(queueId + " was shutdown, rescheduled " + work);
             }
+            NuxeoBlockingQueue q = queuing.getQueue(queueId);
+            if (!q.active) {
+                work.setWorkInstanceState(State.SCHEDULED);
+                queuing.workReschedule(queueId, work);
+                throw new RejectedExecutionException(queueId + " is not active, rescheduled " + work);
```

Comment:
it sounds like an infinite loop, I don't understand why you throw a rejected exception

Reply:
Indeed I am in favor of raising an inconsistent configuration error during start when using memory workmanager impl and disabling processing, this config can only create problems in production:
at best you are losing all scheduled works at worst you are running OOM.